index.html

```diff
@@ -2,192 +2,192 @@
+              <h3>28</h3>
+              <p class="cc-1">Indoor Lanes</p>
+            </div>
           </div>
-        </div>
-        <div class="counter-column">
-          <div class="card">
-            <p><i class="fas fa-tree"></i></p>
-            <h3>26</h3>
-            <p>Outdoor Lanes</p>
+          <div class="counter-column">
+            <div class="card">
+              <p><i class="fas fa-tree f-card"></i></p>
+              <h3>26</h3>
+              <p class="cc-1">Outdoor Lanes</p>
+            </div>
           </div>
-        </div>
-        <div class="counter-column">
-          <div class="card">
-            <p><i class="fas fa-door-closed"></i></p>
-            <h3>2</h3>
-            <p>Private Rooms</p>
+          <div class="counter-column">
+            <div class="card">
+              <p><i class="fas fa-door-closed f-card"></i></p>
+              <h3>2</h3>
+              <p class="cc-1">Private Rooms</p>
+            </div>
           </div>
-        </div>
-        <div class="counter-column">
-          <div class="card">
-            <p><i class="far fa-smile"></i></p>
-            <h3>100+</h3>
-            <p>Happy Customers</p>
+          <div class="counter-column">
+            <div class="card">
+              <p><i class="far fa-smile f-card"></i></p>
+              <h3>100+</h3>
+              <p class="cc-1">Happy Customers</p>
+            </div>
           </div>
         </div>
       </div>
       <!-- end counter section -->
       <!-- start map section -->
       <section class="map-section">
-        <iframe src="https://www.google.com/maps/embed?pb=!1m16!1m12!1m3!1d10847.47646319194!2d-122.44877621259273!3d47.179999114901086!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!2m1!1s528%20Nock%20Point%20Ln%20Tacoma%2C%20WA%2098412!5e0!3m2!1sen!2sus!4v1606113329084!5m2!1sen!2sus" width="600" height="450" frameborder="0" style="border:0;" allowfullscreen="" aria-hidden="false" tabindex="0"></iframe>
-        <div class="map-container">
-          <div class="map-contact">
-            <div class="contact-itm ma-1">
-              <span class="title">Address</span>
-              <span>528 Nock Point Ln
-                <br>
-                Tacoma, WA 98412
-              </span>
-            </div>
-            <div class="contact-itm ma-1">
-              <span class="title">Phone</span>
-              <span><a href="tel:+12535551010">+1 (253)-555-1010</a></span>
-            </div>
-            <div class="contact-itm ma-1">
-              <span class="title">Open Hours</span>
-              <span>Mon-Fri 11 am - 7 pm
-                <br>
-                Sat. 11 am - 6 pm
-                <br>
-                Closed: Sunday
-              </span>
+        <div class="bg-img-map">
+          <div class="map-container">
+            <div class="map-contact">
+              <div class="contact-itm ma-1">
+                <div class="title-width">
+                  <span class="title">Address</span>
```

Comment:
You have a div and a span here, you could probably replace them both with a single header element, say an \<h4> and then target all the h4's in this element to have the same style, width etc.